### PR TITLE
LPS-130597

### PR DESCRIPTION
**Pull request by antonioalmeidab**

### CODE REVIEW

**Comment by alejoceballos** [comment]
@joseabelenda, we are uncertain how to deal with the `PortalException` we are catching, so we are rethrowing it as a `RuntimeException`. I don't think it is the best approach.

Other code samples that we checked out log it or let the original exception be thrown. What is the expected behavior in this particular case?

File: modules/apps/click-to-chat/click-to-chat-web/src/main/java/com/liferay/click/to/chat/web/internal/servlet/taglib/ClickToChatTopHeadJSPDynamicInclude.java [hunk 1]
```diff
@@ -67,137 +66,149 @@ public void include(
...
+				httpServletRequest.setAttribute(
+					ClickToChatWebKeys.CLICK_TO_CHAT_PROVIDER_ACCOUNT_ID,
+					clickToChatProviderAccountId);
+				httpServletRequest.setAttribute(
+					ClickToChatWebKeys.CLICK_TO_CHAT_PROVIDER_ID,
+					clickToChatProviderId);
+			}
+		}
+		catch (PortalException portalException) {
+			throw new RuntimeException(portalException);
```
Comment (alejoceballos) [inline modules/apps/click-to-chat/click-to-chat-web/src/main/java/com/liferay/click/to/chat/web/internal/servlet/taglib/ClickToChatTopHeadJSPDynamicInclude.java:211]: Should we just log this exception? :thinking:

Comment (joseabelenda) [reply modules/apps/click-to-chat/click-to-chat-web/src/main/java/com/liferay/click/to/chat/web/internal/servlet/taglib/ClickToChatTopHeadJSPDynamicInclude.java:211]: @alejoceballos , I think is better throw new SystemException(portalException) instead RuntimeException;
You can do this:
```suggestion
if (_log.isDebugEnabled()) {
     _log.debug(portalException, portalException);
}
 throw new SystemException(portalException);
```

**Review by joseabelenda — APPROVED** [review]
Thanks!

**Review by alejoceballos — APPROVED** [review]
:+1:

**Pull request closed without merging**